lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -70,10 +70,40 @@ sub show : Chained('load') PathPart('')
 sub remove : Local Edit
 {
     my ($self, $c) = @_;
-    my $cdtoc_id  = $c->req->query_params->{cdtoc_id};
-    my $medium_id = $c->req->query_params->{medium_id};
+    my $cdtoc_id  = $c->req->query_params->{cdtoc_id}
+        or $self->error(
+            $c, status => HTTP_BAD_REQUEST,
+            message => l('Please provide a CD TOC ID.')
+        );
+
+    $self->error($c, status => HTTP_BAD_REQUEST,
+                 message => l('The provided CD TOC id is not valid.')
+        ) unless is_database_row_id($cdtoc_id);
+
+    my $medium_id = $c->req->query_params->{medium_id}
+        or $self->error(
+            $c, status => HTTP_BAD_REQUEST,
+            message => l('Please provide a medium ID.')
+        );
+
+    $self->error($c, status => HTTP_BAD_REQUEST,
+                 message => l('The provided medium id is not valid.')
```

Comment:
I guess this was an existing string which had "id" lowercase.

Reply:
Yes, although we could consider uppercasing them.